src/schema.toml

```diff
@@ -71,9 +71,9 @@ theme.notice-background-color = { category = "theme", type = "color", name = "Se
 theme.notice-text-color = { category = "theme", type = "color", name = "Secondary Text Color", description = "[deprecated] Notice and indicator text color.", deprecated="true" }
 theme.notice-widget-color = { category = "theme", type = "color", name = "Secondary Widget Color", description = "[deprecated] Notice and indicator UI widget color.", deprecated="true" }
 theme.favorited-color = { category = "theme", type = "color", name = "Favorited Color", description = "[deprecated] Favorited Color", deprecated = "true" }
-theme.nametag-color = { category = "theme", type = "color", name = "Nametag Color", description = "Avatar nametag color." }
-theme.nametag-text-color = { category = "theme", type = "color", name = "Nametag Color", description = "Avatar nametag text color." }
-theme.themes = { category = "theme", type = "json", name = "Themes", description = "A json array describing custom client themes." }
+theme.nametag-color = { category = "theme", type = "color", name = "Background Color", description="The background color of nametags which float above each room guest." }
```

Comment:
@johnshaughnessy  I don't know enough about this to tell if changing the name corrupts the form?

Reply:
It's OK to change the name, description, etc because those are only used within the admin panel. 

You can't change `theme.nametag-color` without consequence because that's the key that's been written to the database. (You'd have to write code in the client `theme.foo || theme.nametag-color` if you wanted to change it to `theme.foo`, so that existing configs continued to work correctly.)

This ends up in `APP_CONFIG`:
![image](https://user-images.githubusercontent.com/4072106/225129102-89ba48a9-3719-46f1-8204-c975e44756fc.png)

Reply:
@johnshaughnessy  perfect that helps a ton